overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-early-reboot/rhcos-early-reboot.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ -f "/run/ignition-modified-kargs" ] || [ -f "/run/fips-modified" ]; then
+    echo "Rebooting"
+    systemctl --force reboot
```

Comment:
We could split the conditionals and use e.g. `systemctl reboot --message="rhcos-early-reboot: modified kargs" to help admins (and us) debug *why* a reboot happened later (and systemd also nicely groups together the reboot flag with the message in the journal).